app/middleware/performPlaceSearch.js

```diff
@@ -37,9 +37,10 @@ function getCoordinates(place) {
 async function getPlaces(req, res, next) {
   const location = res.locals.location;
   try {
-    let places = await locate.byPlace(location, 100);
+    const allPlaces = await locate.byPlace(location, 100);
+    res.locals.countries = placeHelper.getCountries(allPlaces);
     placeSearches.inc(1);
```

Comment:
I know this hasn't changed but it would be good to have the counter incrementing within the locate module. And the same for the incrementing of the other searches available from `locate` i.e. `byPostcode` and `byLatLon`.